mqtt_client/src/MqttClient.ros2.cpp

```diff
@@ -902,6 +914,100 @@ void MqttClient::isConnectedService(
   response->connected = isConnected();
 }
 
+void MqttClient::newRos2ToMqttService(
+  mqtt_client_interfaces::srv::NewRos2ToMqtt::Request::SharedPtr request,
+  mqtt_client_interfaces::srv::NewRos2ToMqtt::Response::SharedPtr response){
+
+  //@TODO: Implement a response string
+  (void) response; // Avoid compiler warning for unused parameter.
+
+
+  // Add mapping definition to ros2mqtt_
+  Ros2MqttInterface& ros2mqtt = ros2mqtt_[request->ros_topic];

```

Comment:
> The std::map [] operator would return a Ros2MqttInterface& previously set and the code following this snippet would just overwrite the settings. I thought this was the expected behavior, but now that I think about it this code could silently change previously set mappings without warning the user. So I assume it could either be:

> 1) Overwrite mapping settings with new values
> 2) Block existing mapping from being changed if the key is found

> Let me know what you think, please. I think the first option would be better, giving more freedom for the user to change mappings on the fly.

I think it is good to allow to overwrite the settings of existing mappings. I think I was just wondering whether this actually works well and as expected. For example, if I modify the `queue_size`, does this change actually take effect?